server/docker/common-services.yml

```diff
@@ -14,12 +14,15 @@ services:
       - ./../app/:/app/:ro
       - ./../manifestmergetool/app/manifestmergetool.jar:/etc/extender/apps/manifestmergetool.jar:ro
       - ./../configs:/etc/defold/extender:ro
-    entrypoint: ["java","-Xdebug","-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005","-Xmx4g","-XX:MaxDirectMemorySize=2g","-jar","/app/extender.jar"]
+      - ./../../dynamo_home:/dynamo_home:ro
+    entrypoint: ["java","-Xmx4g","-XX:MaxDirectMemorySize=2g","-jar","/app/extender.jar"]
     environment:
-      - EXAMPLE_VAR=1
-      # - DM_DEBUG_COMMANDS=1
-      # - DM_DEBUG_KEEP_JOB_FOLDER=1
-      # - DYNAMO_HOME=/dynamo_home
+      - DYNAMO_HOME
```

Comment:
Define all env vars that can be passed from command line. In case if no env var was passed - docker container doesn't have value for that variable. In case if value of env var was passed via command line - it sets for docker container.